### PR TITLE
chore(templates): add example custom resolver

### PR DESCRIPTION
**Pull request by craicoverflow**
**DRAFT**

This PR adds a custom resolver to the Postgres Backend template.

Opening this as a draft so you can see if this is a good approach. It is pretty much as was discussed in  https://github.com/aerogear/graphback/issues/1637

Once we have agreement I can follow up and add this to the other templates.

### CODE REVIEW

**Review by machi1990 — APPROVED** [review]
Pretty much what we discussed on the issue. I am okay with this approach. 

Once @wtrocki is happy with this too, I am hoping to see the docs updated: they can be based on this example and the codegen trick.

**Comment by wtrocki** [comment]
We need:

- Documentation
- Add this to mongodb as well 
- Use custom method in the client

**Review by wtrocki — APPROVED** [review]

**Review by machi1990 — COMMENTED** [review]
Code LGTM thanks @craicoverflow. I'll need to verify this early tomorrow when I'm infront of my workstation.

File: integration/package.json [hunk 1]
```diff
@@ -17,7 +17,6 @@
   },
   "devDependencies": {
     "@graphql-tools/graphql-file-loader": "6.1.0",
-    "apollo-boost": "0.4.9",
```
Comment (machi1990) [inline integration/package.json:20]: Good catch!

**Review by machi1990 — APPROVED** [review]

**Pull request merged**